udp-ballerina/Module.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+
+This module provides an implementation for sending/receiving datagrams to/from another application process (local or remote) using UDP.
+
+When the local host address or the IP address is not given to the optional `localHost` field, `localhost` address is bound by default.
+The Port number of the local port which is used to connect to the remote server is determined randomly. (ephemeral port)
+
+#### Client
+The `udp:Client` is used to interact with the remote UDP host and it can be defined as follows:
+
+```ballerina
+udp:Client socketClient = check new;
+
+udp:Datagram datagram = {
+    remoteHost: "localhost",
+    remotePort : 48829,
+    data : "Hello Ballerina".toBytes()
+};
+
+check socketClient->sendDatagram(datagram);
+
+readonly & udp:Datagram result = check socketClient->receiveDatagram();
+
+check socketClient->close();
+```
+
+#### ConnectClient
+The `udp:ConnectClient` is configured by providing `remoteHost` and `remotePort`; so that it only receives data from, and sends data to, the configured remote host. Once connected, data may not be received from or sent to any other hosts. The client remains connected until it is explicitly it is closed.
```

Comment:
```suggestion
The `udp:ConnectClient` is configured by providing the `remoteHost` and `remotePort` so that it only receives data from and sends data to the configured remote host. Once connected, data may not be received from or sent to any other hosts. The client remains connected until it is explicitly closed.
```